web/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "APD app for CMS HITECH program",
   "main": "app.js",
   "scripts": {
-    "storybook": "start-storybook -p 9001 -c .storybook",
+    "storybook": "start-storybook --ci -p 9001 -c .storybook",
```

Comment:
We went from storybook 3 to 4.  They changed it so the `start-storybook` command launches your default browser unless you use this `--ci` option.  So to keep things basically the same for us, I added the flag.  But also I hate when CLI tools open my browser. 🤷‍♂️